parser_test.go

```diff
@@ -180,6 +194,32 @@ func TestParser_ParseGeneralApiInfoTemplated(t *testing.T) {
 	assert.Equal(t, expected, string(b))
 }
 
+func TestParser_ParseGeneralApiInfoExtensions(t *testing.T) {
+	// should be return an error because extension value is not a valid json
+	func() {
```

Comment:
why needs enclosed with `func()`?

Reply:
just a pattern to exclude side effect from other tests